packages/ERTP/docs/INPUT_VALIDATION.md

```diff
@@ -2,7 +2,7 @@
 
 ## Requirements
 
-We want ERTP to be robust against malicious code in the same vat, so the ERTP must be robust without the kind of checks that we get for free in intervat communication going through `@endo/marshal` and `@agoric/swingset-vat`. In other words, the only tools at our disposal are being in a Hardened JS environment, and helpers that we import and call directly, such as `passStyleOf` from `@endo/marshal`.
+We want ERTP to be robust against malicious code in the same vat, so the ERTP must be robust without the kind of checks that we get for free in intervat communication going through [@endo/marshal](https://www.npmjs.com/package/@endo/marshal) and [@agoric/swingset-vat](../../SwingSet). In other words, the only tools at our disposal are being in a Hardened JS environment, and helpers that we import and call directly, such as `passStyleOf` from `@endo/marshal`.
```

Comment:
Did maybe your regexps not take into account multiple package mentions on the same line, or is it by design that the last occurrence of `@endo/marshal` on this line is not hyperlinked?

Reply:
This one was intentional.